reddit.js

```diff
@@ -2,4 +2,49 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+var children = reddit.data.children;
+// var childTitles = children[0].data.title;
+// var childLinks = children[0].data.permalink;
+// var childText = children[0].data.selftext;
+// var points = children[0].data.score;
+
+// console.log(childTitles);
+
+
+// QUESTION 1
+children.forEach(function(titles) {
+	console.log('Title: ' + titles.data.title);
+});
+
+// QUESTION 2
+var links = children.map(function(link) {
+	return link.data.permalink;
+});
+
+links.forEach(function(url) {
```

Comment:
Now that we've prepended ```'http://reddit.com'``` to the return statement above, we no longer need this function here.